src/imex.rs

```diff
@@ -484,12 +484,12 @@ async fn import_backup(
 /// Returns Ok((temp_db_path, temp_path, dest_path)) on success. Unencrypted database can be
 /// written to temp_db_path. The backup can then be written to temp_path. If the backup succeeded,
 /// it can be renamed to dest_path. This guarantees that the backup is complete.
-fn get_next_backup_path(folder: &Path, backup_time: i64) -> Result<(PathBuf, PathBuf, PathBuf)> {
+fn get_next_backup_path(folder: &Path, addr: &str, backup_time: i64) -> Result<(PathBuf, PathBuf, PathBuf)> {
     let folder = PathBuf::from(folder);
     let stem = chrono::NaiveDateTime::from_timestamp_opt(backup_time, 0)
         .context("can't get next backup path")?
         // Don't change this file name format, in `dc_imex_has_backup` we use string comparison to determine which backup is newer:
-        .format("delta-chat-backup-%Y-%m-%d")
+        .format(&format!("delta-chat-{}-%Y-%m-%d", &addr).to_string())
```

Comment:
as the comment say, we need to adapt `has_backup()` as well (even though it is fading out somehow). otherwise, it  won't return any backup of alice@foo.bar if there is another account with a "higher" letter (as bob@foo.bar)

quick fix would be to change `name > newest_backup_name` to the rust-equivalent of `name.right(17) > newest_backup_name.right(17)`

Reply:
> Instead of adapting `has_backup()` maybe do similar to #4737 and allow to select a file instead of folder?

I don't know the details, but that is how it already works in DC in recent android versions, you select the backup file directly, not a folder, the folder is only used on older android, same for key importing

Reply:
> Regarding the filename format, I would move the address to the end, so sorting by filename shows the newest files grouped.

good idea - that way, we probably do not need to adapt has_backup() and are compatible to older versions

(has_backup() is used at least on ios still)

Reply:
I moved the address after the date as recommended,

btw, maybe the self-address needs some sanitizing to avoid issues in some filesystems's invalid characters rules? feel free to take over this PR and do modifications

Reply:
the running number should better go before the address as well to ensure correct sorting and compatibility with previous cores.

Reply:
Btw, the loop choosing the running number looks not nice, i'd better add hh-mm-ss right after the date

Reply:
> Btw, the loop choosing the running number looks not nice, i'd better add hh-mm-ss right after the date

this is out of scope of this pr. iirc, the idea of index instead of time is to make filenames a bit shorter. and as one does also not do backups every few seconds. also "second backup" on monday may be easier to memorize than exact time. we also got never user discussions here afaik

Reply:
i added some commits to adapt documentation and to add the address to the very end to maintain sortability